### PR TITLE
moreutils 0.59

### PR DESCRIPTION
**Pull request by nijikon**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/moreutils.rb [hunk 1]
```diff
@@ -22,15 +22,13 @@ class Moreutils < Formula
   conflicts_with "task-spooler", :because => "Both install a 'ts' executable."
 
   resource "Time::Duration" do
-    url "https://cpan.metacpan.org/authors/id/A/AV/AVIF/Time-Duration-1.1.tar.gz"
```
Comment (DomT4) [inline Formula/moreutils.rb]: Please don't remove the `https://` URLs here, they still work with the update:

```
https://cpan.metacpan.org/authors/id/N/NE/NEILB/Time-Duration-1.20.tar.gz
```

File: Formula/moreutils.rb [hunk 2]
```diff
@@ -22,15 +22,13 @@ class Moreutils < Formula
...
-    mirror "http://search.cpan.org/CPAN/authors/id/A/AV/AVIF/Time-Duration-1.1.tar.gz"
-    sha256 "a69c419c4892f21eba10002e2ab8c55b657b6691cf6873544ef99ef5fd188f4e"
+    url "http://search.cpan.org/CPAN/authors/id/N/NE/NEILB/Time-Duration-1.20.tar.gz"
+    sha256 "458205b528818e741757b2854afac5f9af257f983000aae0c0b1d04b5a9cbbb8"
   end
 
   resource "IPC::Run" do
-    url "https://cpan.metacpan.org/authors/id/T/TO/TODDR/IPC-Run-0.92.tar.gz"
```
Comment (DomT4) [inline Formula/moreutils.rb]: Same notation as above.

**Comment by nijikon** [comment]
@DomT4 sure, sorry for that. I've tweaked audit to handle this better.

**Comment by DomT4** [comment]
What was the audit saying/doing prior to the tweak? Seems okay here now, although it's nice to avoid `ftp` where possible because firewalls can be prone to closing that port.

**Comment by nijikon** [comment]
Nothing. That's why I tweaked it. Maybe we could allow http in mirrors and add an audit that we prefer http over ftp in strict mode?

**Comment by DomT4** [comment]
Sorry, you've lost me a bit here, which albeit is quite easy at 5am 😄. If `https://` is available it should be used. Ideally mirrors should be using a different website because if one protocol is down it's not regularly an unsafe bet others will be dead as well.

Was something amiss with the usual `http://search.cpan.org/CPAN/authors` mirrors we use for Perl stuff?

**Comment by nijikon** [comment]
No worries 😄  I was just wondering if we could tweak audit a bit for this. So for ```url``` we would allow https or http only no ftp. For this particular case we can make it ```http://search.cpan.org/CPAN/authors``` but than I would have to change this https://github.com/Homebrew/brew/commit/7bbcf9cacc18aabf78a21ec5cc5892d440631e08 so it would only work on url not on mirror. What do you think?

**Comment by DomT4** [comment]
https://github.com/Homebrew/brew/pull/302

**Comment by DomT4** [comment]
👍 :shipit:.

**Pull request closed without merging**